pkg/safeguards/preprocessing/preprocessing.go

```diff
@@ -97,31 +84,21 @@ func RenderKustomizeManifest(kustomizationPath, tempDir string) ([]safeguards.Ma
 	kustomizeFS := filesys.MakeFsOnDisk()
 	resMap, err := k.Run(kustomizeFS, kustomizationPath)
 	if err != nil {
-		return nil, fmt.Errorf("Error building manifests: %s\n", err.Error())
+		return nil, fmt.Errorf("error building manifests: %s", err.Error())
 	}
 
 	// Output the manifests
-	var manifestFiles []safeguards.ManifestFile
-	kindMap := make(map[string]int)
+	var manifestFiles []sgTypes.ManifestFile
 	for _, res := range resMap.Resources() {
 		yamlRes, err := res.AsYAML()
 		if err != nil {
-			return nil, fmt.Errorf("Error converting resource to YAML: %s\n", err.Error())
-		}
-
-		// index of every kind of manifest for outputRenderPath
-		kindMap[res.GetKind()] += 1
-		outputRenderPath := filepath.Join(tempDir, strings.ToLower(res.GetKind())) + fmt.Sprintf("-%d.yaml", kindMap[res.GetKind()])
-
-		err = kustomizeFS.WriteFile(outputRenderPath, yamlRes)
-		if err != nil {
-			return nil, fmt.Errorf("Error writing yaml resource: %s\n", err.Error())
+			return nil, fmt.Errorf("error converting resource to YAML: %s", err.Error())
 		}
 
 		// write yamlRes to dir
-		manifestFiles = append(manifestFiles, safeguards.ManifestFile{
-			Name: res.GetName(),
-			Path: outputRenderPath,
+		manifestFiles = append(manifestFiles, sgTypes.ManifestFile{
+			Name:            res.GetName(),
```

Comment:
nit: weird spacing here

Reply:
Looks like that's the correct spacing because it needs to follow the spacing of the next line `ManifestContent: yamlRes` on 101